packages/components/src/flex/types.ts

```diff
@@ -79,7 +83,7 @@ export type FlexItemProps = {
 	/**
 	 * The children elements.
 	 */
-	children: React.ReactNode;
+	children: ReactNode;
```

Comment:
I think we should mark this as optional, because FlexItem/FlexBlock doesn't necessarily need `children` if used with the `as` prop. Like `<FlexItem as={ Spinner } />`.

```suggestion
	children?: ReactNode;
```

Looking at the original PR (#31817), it says:

> The `children` prop is added to all components that require it explicitly

so I _think_ it's safe to make optional, given that it isn't explicitly required here. (cc @ciampo just to double check)

Reply:
Yeah. Since #31817, `WordPressComponentProps` doesn't include the `children` prop anymore — and so a component that needs `children` needs to list that prop explicitly. I'm good with the proposal of marking this prop optional